README.rst

```diff
@@ -52,6 +52,7 @@ Installing these plugins will add extra functionality to the language server:
 * pyls-mypy_ Mypy type checking for Python 3
 * pyls-isort_ Isort import sort code formatting
 * pyls-black_ for code formatting using Black_
+* pyls-memestra_ a static analysis tool for Python, which detects the use of deprecated APIs.
```

Comment:
I think that it might be shortened to better fit with the others which all have 5-6 words in descriptions:

```suggestion
* pyls-memestra_ for detecting the use of deprecated APIs
```

My thinking is that all of these are for Python and all do static analysis so it might be fine to leave this out.

Reply:
True! Thank you.